sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/CxxPlugin.java

```diff
@@ -500,6 +513,15 @@
         .subCategory(subcateg)
         .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
         .index(5)
+        .build(),
+      PropertyDefinition.builder(LANG_PROP_PREFIX
+        + UnitTestConfiguration.NUNIT_TEST_RESULTS_PROPERTY_KEY)
+        .multiValues(true)
+        .name("NUnit Test Reports Paths")
+        .description("Example: \"TestResult.xml\", \"TestResult1.xml,TestResult2.xml\" or \"C:/TestResult.xml\"")
```

Comment:
see https://github.com/SonarSource/sonar-csharp/blob/master/sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpUnitTestResultsProvider.java

* should the property become public?
* should it stay hidden?
* should we remove the support completely?